deployments/engine/docker-compose/1m1e_with_s3.yaml

```diff
@@ -0,0 +1,104 @@
+version: "2.3"
```

Comment:
Why do we need this config file?

Reply:
it's my local benchmark config. Maybe other can also re-use it